l5kit/l5kit/rasterization/rasterizer_builder.py

```diff
@@ -131,6 +131,18 @@ def build_rasterizer(cfg: dict, data_manager: DataManager) -> Rasterizer:
     filter_agents_threshold = raster_cfg["filter_agents_threshold"]
     history_num_frames = cfg["model_params"]["history_num_frames"]
 
+    if "history_num_frames_to_rasterize" in raster_cfg:
+        _override_val = raster_cfg["history_num_frames_to_rasterize"]
```

Comment:
I would put a warning here
"you're overwriting history_num_frames in the rasteriser only, image and history_positions will have different lengths at runtime"

This way we have at least a warning when the forward will fail xD